### PR TITLE
Readme revisions

### PR DESCRIPTION
**Pull request by pgale61**
@maratkomarov @rtfee Submitting this for initial review of the README. Not ready to merge. Plan is to merge at the same time as the blog posts go live and the READ requires a link to those posts. Also need to do a separate PR to add more descriptive comments to each policy to aid understanding.

### CODE REVIEW

**Comment by rtfee** [comment]
@pgale61 anyone outside of Scalr will not be able to create a branch in our repo. They should fork the project, create their changes in their own repo, and then open a PR from that repo against our master.

**Comment by maratkomarov** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,5 +1,124 @@
...
 
 [![OPA tests](https://github.com/Scalr/sample-tf-opa-policies/workflows/OPA/badge.svg)](https://github.com/Scalr/sample-tf-opa-policies/actions?query=workflow%3AOPA)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/Scalr/sample-tf-opa-policies/pulls)
+[![Documentation Status](https://readthedocs.com/projects/scalr-athena/badge/?version=latest)](https://docs.scalr.com/en/latest/opa.html)
+
+A sample collection of OPA policies to test against Terraform run in Scalr.
+
+## Resources
+
+* Blog series for getting started with OPA for Terraform in Scalr : https://scalr.com//blog/opa-series-part-1-open-policy-agent-and-terraform/
```
Comment (petroprotsakh) [inline README.md:11]: Extra slash here - `scalr.com//blog`.
(Is the article not published yet? The link goes to 404 atm)

Comment (petroprotsakh) [reply README.md:11]: I also find it a bit weird to have a whitespace before colon throughout the document, but I'm not sure it is a rule in English punctuation

File: README.md [hunk 2]
```diff
@@ -1,5 +1,124 @@
...
+```
+
+## Policies
+
+Summary decscriptions of each policy. Detailed descriptions of each rule can be found as comments in the policy file.
+Many policies contain arrays of values that are checked against resources. The arrays and reason messages are of course customisable.
+
+| Policy                                 | Description                                                              |
+| -------------------------------------- | ------------------------------------------------------------------------ |  
+| [aws/enforce_aws_iam_and_workspace.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_aws_iam_and_workspace.rego) | Checks valid IAM roles for provder and workspace.                        |
```
Comment (petroprotsakh) [inline README.md:42]:
```suggestion
| [aws/enforce_aws_iam_and_workspace.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_aws_iam_and_workspace.rego) | Checks valid IAM roles for provider and workspace.                        |
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,124 @@
...
+| [aws/enforce_aws_resource.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_aws_resource.rego) | Check resource types against an allowed list.                            |
+| [aws/enforce_ebs_del_on_term.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_ebs_del_on_term.rego) | Check `delete_on_termination = true` is set for EBS volumes.             |
+| [aws/enforce_iam_instance_profiles.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_iam_instance_profiles.rego) | Check IAM instance profiles is in allowed list.                          |
```
Comment (petroprotsakh) [inline README.md:45]:
```suggestion
| [aws/enforce_iam_instance_profiles.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_iam_instance_profiles.rego) | Check IAM instance profile is in allowed list.                          |
```

File: README.md [hunk 4]
```diff
@@ -1,5 +1,124 @@
...
+| [aws/enforce_instance_subnets.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_instance_subnets.rego) | Check instances are using allowed subnets |
+| [aws/enforce_kms_key_names.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_kms_key_names.rego) | Check KMS keys (by name) against allowed list.                           |
+| [aws/enforce_lb_subnets.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_lb_subnets.rego) | Check Loadbalancers are using allowed subnets |
+| [aws/enforce_s3_buckets_encryption.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_s3_buckets_encryption.rego) | Check encryption is set for S3 buckets.                                  |
+| [aws/enforce_sec_group.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_sec_group.rego) | Check security groups have been sepcified and are in allowed list.       |
```
Comment (petroprotsakh) [inline README.md:50]:
```suggestion
| [aws/enforce_sec_group.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_sec_group.rego) | Check security groups have been specified and are in allowed list.       |
```

File: README.md [hunk 5]
```diff
@@ -1,5 +1,124 @@
...
+| [aws/enforce_rds_subnets.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_rds_subnets.rego) | Check RDS clsuters are using allowed subnets |
```
Comment (petroprotsakh) [inline README.md:51]:
```suggestion
| [aws/enforce_rds_subnets.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/aws/enforce_rds_subnets.rego) | Check RDS clusters are using allowed subnets |
```

File: README.md [hunk 6]
```diff
@@ -1,5 +1,124 @@
...
+| [modules/pin_module_version.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/modules/pin_module_version.rego) | Enforces use of specific module versions.                                |
+| [modules/required_modules.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/modules/required_modules.rego) | Checks resources are only be created via specific modules.               |
+| [placement/cloud_location.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/placement/cloud_location.rego) | Checks resources are deployed to specific regions in each cloud.         |
+| [providers/blacklist_provider.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/providers/blacklist_provider.rego) | Implements a provider blacklist.                                         |
+| [user/user.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/user/user.rego) | Restricts which users can trigger terrafrom runs. Works for CLI and VCS. |
```
Comment (petroprotsakh) [inline README.md:65]:
```suggestion
| [user/user.rego](https://github.com/Scalr/sample-tf-opa-policies/blob/master/user/user.rego) | Restricts which users can trigger terraform runs. Works for CLI and VCS. |
```

File: README.md [hunk 7]
```diff
@@ -1,5 +1,124 @@
...
+If you submit a new policy you must include the following files.
+
+* The `*.rego` file with the policy code.
+* `*.mock.json` containing test data mocks. You should inlcude data for both valid and invalid evaluation of each rule in the policy,.
+* `*.test.rego` defining the tests to be run and expected results when the PR checks are performed.
+
+To submit a PR follow the standard process.
+
+1. Fork the repo
+2. Clone locally and creaete a new branch
```
Comment (petroprotsakh) [inline README.md:103]:
```suggestion
2. Clone locally and create a new branch
```

File: README.md [hunk 8]
```diff
@@ -1,5 +1,124 @@
...
+3. Commit and push
+4. Submit PR request
```
Comment (petroprotsakh) [inline README.md:105]: Submit PR
or
Submit Pull Request

File: README.md [hunk 9]
```diff
@@ -1,5 +1,124 @@
...
+
+```bash
+$ terraform plan --out planfile
+$ terraform show -json planfile > plan.json
+$ opa eval --format pretty --data policy.rego -i plan.json data.terraform.deny
+```
+
+## Policies
+
+Summary decscriptions of each policy. Detailed descriptions of each rule can be found as comments in the policy file.
```
Comment (petroprotsakh) [inline README.md:37]:
```suggestion
Summary descriptions of each policy. Detailed descriptions of each rule can be found as comments in the policy file.
```

File: README.md [hunk 10]
```diff
@@ -1,5 +1,124 @@
...
+* Sample `terraform plan` data the policy should work against.
+
+### Pull Requests
+
+Better still have a go at fixing bug or implementing new policy examples yourself and submit a Pull Request.
+
+If you submit a new policy you must include the following files.
+
+* The `*.rego` file with the policy code.
+* `*.mock.json` containing test data mocks. You should inlcude data for both valid and invalid evaluation of each rule in the policy,.
```
Comment (petroprotsakh) [inline README.md:97]:
```suggestion
* `*.mock.json` containing test data mocks. You should include data for both valid and invalid evaluation of each rule in the policy.
```

**Review by petroprotsakh — APPROVED** [review]
LGTM, just highlighted several typos

File: README.md (hunk 1 above)
Comment (pgale61) [reply README.md:11]: Will fix / and :

File: README.md (hunk 8 above)
Comment (pgale61) [reply README.md:105]: Will fix

**Pull request merged**